backends/cuda/ceed-cuda.h

```diff
@@ -113,6 +113,7 @@ typedef struct {
   CeedVector *qvecsout;   /// Output Q-vectors needed to apply operator
   CeedInt    numein;
   CeedInt    numeout;
+  bool e_and_q_layout_diff;
```

Comment:
The style in the CUDA backends would be to use `eandqlayoutdiff`  or maybe `eandqdiffer`?

Reply:
I'm not a very consistent style person (unfortunately), so you can unify the way you prefer.

Reply:
I like `eandqdiffer`, I think I will go with that.